certificate/views.py

```diff
@@ -1110,7 +1110,41 @@ def osdag_workshop_download(request):
     return render_to_response('osdag_workshop_download.html', context, ci)
 
 def osdag_workshop_feedback(request):
-    return render_to_response('osdag_workshop_feedback.html')
+    context = {}
+    ci = RequestContext(request)
+    form = FeedBackForm()
+    questions = Question.objects.filter(purpose='OWS')
+    if request.method == 'POST':
+        form = FeedBackForm(request.POST)
+        if form.is_valid():
+            data = form.cleaned_data
+            try:
+                FeedBack.objects.get(email=data['email'].strip(), purpose='DWS')
```

Comment:
Here purpose should be "OWS" as it is for Osdag Workshop and not "DWS".
All feedbacks will go under DWS!
Kindly test your code thoroughly and push.